packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -3260,36 +3370,22 @@ export default abstract class SqlIntegration
       )
     );
 
-    if (params.dimensions.length > 1) {
-      throw new Error(
-        "Multiple dimensions not supported in metric analysis yet. Please contact GrowthBook."
-      );
-    }
-    const dimension = params.dimensions[0];
-    let dimensionCol = this.castToString("''");
-    if (dimension?.type === "experiment") {
-      dimensionCol = `dim_exp_${dimension.id}`;
-    } else if (dimension?.type === "user") {
-      dimensionCol = `dim_unit_${dimension.dimension.id}`;
-    } else if (dimension?.type === "date") {
-      dimensionCol = `${this.formatDate(
-        this.dateTrunc("first_exposure_timestamp")
-      )}`;
-    } else if (dimension?.type === "activation") {
-      dimensionCol = this.ifElse(
-        `first_activation_timestamp IS NULL`,
-        "'Not Activated'",
-        "'Activated'"
-      );
-    }
+    // ensure always at least length 1 with plain dim
+    const dimensionCols = params.dimensions.map((d) => this.getDimensionCol(d));
```

Comment:
Is the comment here true? Where is it enforcing that `params.dimensions.length > 0`?